README.md

```diff
@@ -261,7 +265,19 @@ Text / Knowledge-base data can be nested in queries. This allows queries like on
 For now, each text-record variable is required to have a triple `ql:contains-word/entity WORD/URI`. 
 Pure connections to variables (e.g. "Books with a description that mentions a plant.") are planned for the future.
 
+To obtain a list of available relations and their counts `ql:has-relation` can be used, if the index has a patterns file, and the server was started with the --patterns option:
```

Comment:
can you add information about how such a patterns file is obtained (IndexBuilder option/format)